### PR TITLE
360 video playback sample fixes

### PR DESCRIPTION
**Pull request by polszta**

### CODE REVIEW

File: Samples/360VideoPlayback/cpp/360VideoPlayback.vcxproj [hunk 1]
```diff
@@ -217,18 +217,30 @@
       <ShaderModel Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">5.0</ShaderModel>
       <ShaderModel Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">5.0</ShaderModel>
       <ShaderType Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">Geometry</ShaderType>
+      <ShaderType Condition="'$(Configuration)|$(Platform)'=='Release|x64'">Geometry</ShaderType>
```
Comment (MikeRiches) [inline Samples/360VideoPlayback/cpp/360VideoPlayback.vcxproj]: Were these changes intended to be part of the pull request?

File: Samples/360VideoPlayback/cpp/Content/VideoRenderer.cpp [hunk 2]
```diff
@@ -97,18 +97,17 @@ void VideoRenderer::Render()
...
-        // VPAndRTArrayIndexFromAnyShaderFeedingRasterizer optional feature,
-        // a pass-through geometry shader is used to set the render target
-        // array index.
-        context->GSSetShader(
-            m_geometryShader.Get(),
-            nullptr,
-            0
-        );
-    }
+    // On devices that do not support the D3D11_FEATURE_D3D11_OPTIONS3::
```
Comment (MikeRiches) [inline Samples/360VideoPlayback/cpp/Content/VideoRenderer.cpp]: It would be more clear, and potentially safer as well, to have an if-else case where the else case sets nullptr directly instead of relying on m_geometryShader.get() to be nullptr.

File: Samples/360VideoPlayback/cpp/360VideoPlayback.vcxproj (hunk 1 above)
Comment (polszta) [reply Samples/360VideoPlayback/cpp/360VideoPlayback.vcxproj]: Yes. As it turns out, the sample would fail to build on Release configurations, preventing me from fully testing the fix.

File: Samples/360VideoPlayback/cpp/Content/VideoRenderer.cpp (hunk 2 above)
Comment (polszta) [reply Samples/360VideoPlayback/cpp/Content/VideoRenderer.cpp]: Applied your suggestion.

**Comment by oldnewthing** [comment]
Thanks. Propagated fix into C# sample and pulled into next refresh.
